### PR TITLE
11 multiple institutions

### PR DESCRIPTION
**Pull request by dankaiser1808**
Some Markers were overlapping so that a lot of institution names could not be seen. Overlapping institution names will now be summarized in a single marker. Please be as nitpicky as possible.

### CODE REVIEW

File: generate-map.py [hunk 1]
```diff
@@ -48,6 +50,42 @@ def getGeoCode(country, city, organization, dataList):
         return geoLocation
 
 
+def fixOverlappingEntries():
```
Comment (lkiesow) [inline generate-map.py:51]: Suggestion:

```python
locations = {}
for organization in data:
    key = (organization['longitude'], organization['latitude'])
    # check if we already encountered this location
    if locations.get(key):
        locations[key]['organization'] += ', ' + organization['organization']
    else:
        locations[key] = organization
return locations.values()
```

File: leafletmap.js [hunk 2]
```diff
@@ -1,26 +1,26 @@
 window.onload = function () {
     console.log("Script is running")
     var basemap = L.tileLayer('https://a.tile.openstreetmap.org/{z}/{x}/{y}.png', {
-		attribution: '&copy; <a href="https://osm.org/copyright">OpenStreetMap</a> contributors',
```
Comment (lkiesow) [inline leafletmap.js]: The changes to HTML and JavaScript are not related to this pull request. Plus, the JavaScript changes break the indention by mixing tabs and spaces.

Comment (lkiesow) [reply leafletmap.js]: Sorry, this actually removes the tabs and replaces it with spaces only. My mistake.
Still, this change has nothing to do with the topic of this pull request ;-)

File: generate-map.py (hunk 1 above)
Comment (dankaiser1808) [reply generate-map.py:51]: I added the suggestions. Your way is much more easy and clear. thank you! :D

File: leafletmap.js (hunk 2 above)
Comment (dankaiser1808) [reply leafletmap.js]: I removed the changes in the HTML and js file.

**Comment by Rillke** [comment]
#23 approaches this by adding the cluster plugin which also allows dissolving multiple entries located at the same geographic position on map.

I would be against summarizing multiple (possibly unrelated) institutions into a single marker.

**Pull request closed without merging**